test/integration/pods/pods_test.go

```diff
@@ -155,7 +157,9 @@ func TestPodReadOnlyFilesystem(t *testing.T) {
 	ns := framework.CreateTestingNamespace("pod-readonly-root", s, t)
 	defer framework.DeleteTestingNamespace(ns, s, t)
 
-	client := client.NewOrDie(&restclient.Config{Host: s.URL, ContentConfig: restclient.ContentConfig{GroupVersion: testapi.Default.GroupVersion()}})
```

Comment:
I tried, I can't, the mute thread link does nothing. I think someone added me incorrectly thinking I was you a long time ago. I used to only get a msg once a month so I ignored it, now it qualifies as spam I get so many. I also develop, just not this